### DISCUSSION TITLE
能不能支持私有隧道

### DISCUSSION THREAD

**Discussion by xmsz** [natfrp/wiki#190] [想法] [answered]
背景
- 我们想自己做内网穿透，有自己的frp服务器
- 现在缺少客户端和管理端

使用官方的adminUI，实在太简陋
自己开发，没多大意义

所以如果natfrp可以支持私有隧道，那就和简单了，我只要提高frp服务器就行

对于付费问题，我们愿意按私有隧道数量进行付费

**Comment by fakeboboliu** [comment, marked as answer]
感谢您对我们系统的认可和兴趣，
因为目前的系统设计主要服务于我们的 2C 平台功能，我们暂时并无提供私有部署的计划。

**Reply by xmsz** [reply]
ok 期待一下
